<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>QmidiCtl - A MIDI remote controller via IP/UDP multicast.

   Copyright (C) 2010-2025, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qmidictlMainForm</class>
 <widget class="QMainWindow" name="qmidictlMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="qmidictl.qrc">:/images/qmidictl.svg</iconset>
  </property>
  <widget class="QMenuBar" name="MainMenuBar">
   <addaction name="optionsAction"/>
   <addaction name="configureAction"/>
   <addaction name="aboutAction"/>
   <addaction name="exitAction"/>
  </widget>
  <widget class="QWidget" name="MainCentralWidget">
   <layout class="QVBoxLayout" name="MainCentralLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QGridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="midiInTextLabel">
          <property name="text">
           <string>MIDI In</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="midiInLedLabel">
          <property name="pixmap">
           <pixmap resource="qmidictl.qrc">:/images/ledOff.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="midiOutTextLabel">
          <property name="text">
           <string>Out</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="midiOutLedLabel">
          <property name="pixmap">
           <pixmap resource="qmidictl.qrc">:/images/ledOff.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="qmidictlMixerStrip" name="strip1" native="true"/>
        </item>
        <item>
         <widget class="qmidictlMixerStrip" name="strip2" native="true"/>
        </item>
        <item>
         <widget class="qmidictlMixerStrip" name="strip3" native="true"/>
        </item>
        <item>
         <widget class="qmidictlMixerStrip" name="strip4" native="true"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QDial" name="jogWheelDial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>220</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>RS&amp;T</string>
          </property>
          <property name="icon">
           <iconset resource="qmidictl.qrc">:/images/formReset.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rewindButton">
          <property name="text">
           <string>RE&amp;W</string>
          </property>
          <property name="icon">
           <iconset resource="qmidictl.qrc">:/images/formRewind.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="text">
           <string>&amp;PLAY</string>
          </property>
          <property name="icon">
           <iconset resource="qmidictl.qrc">:/images/formPlay.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="text">
           <string>&amp;STOP</string>
          </property>
          <property name="icon">
           <iconset resource="qmidictl.qrc">:/images/formStop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recordButton">
          <property name="text">
           <string>&amp;REC</string>
          </property>
          <property name="icon">
           <iconset resource="qmidictl.qrc">:/images/formRecord.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="forwardButton">
          <property name="text">
           <string>&amp;FFWD</string>
          </property>
          <property name="icon">
           <iconset resource="qmidictl.qrc">:/images/formForward.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QPushButton" name="prevStripPageButton">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="nextStripPageButton">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="optionsAction">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="configureAction">
   <property name="text">
    <string>&amp;Configure</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qmidictlMixerStrip</class>
   <extends>QWidget</extends>
   <header>qmidictlMixerStrip.h</header>
  </customwidget>
 </customwidgets>
 <layoutdefault spacing="8" margin="8"/>
 <tabstops>
  <tabstop>prevStripPageButton</tabstop>
  <tabstop>nextStripPageButton</tabstop>
  <tabstop>jogWheelDial</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>rewindButton</tabstop>
  <tabstop>playButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>recordButton</tabstop>
  <tabstop>forwardButton</tabstop>
 </tabstops>
 <resources>
  <include location="qmidictl.qrc"/>
 </resources>
 <connections/>
</ui>
